theme.js

```diff
@@ -3055,4 +3060,20 @@ export default (variables = defaultThemeVariables) => ({
       opacity: 0.3,
     },
   },
+
+  'shoutem.ui.DateTimePicker': {
+    buttonContainer: {
+      width: 50,
+      height: 50,
+      backgroundColor: '#222222',
+    },
+    icon: { color: '#FFFFFF', height: 30, width: 30 },
```

Comment:
AFAIK `Icon` receives color via `fill={color}`, not via style prop.

I've been writing it as:
`iconFill: '#FFFFFF'`

Then using that in the icon as `fill={style.iconFill}`.

If there's multiple icons, I gave the style properties different names, e.g. `confirmIcon` and `confirmIconFill`.

Reply:
Doesn't work for me.
It's because `Icon` component from `shoutem/ui` sets `style.color` as `fill` param.